app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectPresenter.kt

```diff
@@ -143,31 +142,37 @@ class DeviceConnectPresenter(deviceActivity: DeviceActivity, manager: WifiManage
 
     override fun onSendCredentialSuccess() {
         Timber.d("WIFI - SUCCESSFUL")
-        deviceConnectView?.onDeviceConnectionSuccess()
+        deviceConnectView?.onDeviceConnectionSuccess(utilModel.getString(R.string.wifi_success))
+        makeAuthRequest()
     }
 
     override fun onSendCredentialFailure() {
         Timber.d("WIFI - FAILURE")
-        //   deviceConnectView?.onDeviceConnectionError("Wifi Cred Failure", "Not done properly")
+        deviceConnectView?.stopProgress()
+        deviceConnectView?.onDeviceConnectionError("Wifi Cred Failure", "Not done properly")
```

Comment:
Cred ---> credentials, small f in Failure

Reply:
This is a basic connection, : 
as mario said on gitter first the base should be done and then modified, these strings won't even exist afterwards but right now these are important just to symbolize some work when we connect to the device.